lms/djangoapps/discussion/templates/discussion/discussion_profile_page.html

```diff
@@ -1,11 +1,9 @@
 ## mako
 
-<%! main_css = "style-discussion-main" %>
+<%namespace name='static' file='../static_content.html'/>
 
 <%page expression_filter="h"/>
-<%inherit file="../main.html" />
```

Comment:
Can you explain why you removed this? The main.html file imports many of the features that exist on all pages, header, footer, etc.

Reply:
Oh just kidding, that was because you made it a fragment, disregard ^